runtimes/pythonrt/py-sdk/autokitteh/events.py

```diff
@@ -18,7 +18,9 @@ def unsubscribe(subscription_id: str) -> None:
     pass
 
 
-def next_event(subscription_id: str | list, *, timeout: timedelta = None) -> AttrDict:
+def next_event(
+    subscription_id: str | list[str], *, timeout: timedelta | int = None
```

Comment:
Why int and not float? I can't do a 500ms timeout.

Reply:
right! will fix

Reply:
see https://github.com/autokitteh/autokitteh/pull/815